API/Controllers/CountryController.cs

```diff
@@ -29,6 +29,51 @@ public IEnumerable<CountryContract> GetAllCountries()
             return countryContracts;
         }
 
+        [HttpGet()]
+        public IEnumerable<string> GetAllCountryNames()
+        {
+            var countries = _countryRepository.GetAllCountries();
```

Comment:
Wouldn't it be better to get all names form the database in the repository?

Reply:
Removed